### ISSUE TITLE
Visual star search replace

### ISSUE THREAD

**Issue by 01556** [bronson/vim-visual-star-search#12]
I request adding an option (if not already present) to use this plugin as normal, but hit `?` or a similar key to start replacing text. For instance, say I have a file:
```
......occurrence...
....occurrence...
```
I would then select the first `occurrence` and hit `*` as usual, but press `?` and type what to replace it with (then hit enter)

**Comment by bronson** [comment]
Yeah, this would be handy. I've always switched out of search mode and written a :substitution expression...  and I've never questioned doing that until now.

I would expect the solution to apply to all star-searches, not just visual-star-searches. Heck, I'd expect it to work any time `n` and `p` take you to a new match, so _all_ searches.

Have you looked for a plugin that can perform find-and-replace using vim's built-in searching (without resorting to writing ex :s// expressions)? If that plugin exists, it should work fine with visual-star-search.

If it doesn't, someone should write it. :)
